skrub/_table_vectorizer.py

```diff
@@ -474,6 +553,78 @@ def _clone_transformers(self):
 
         # TODO: check that the provided transformers are valid
 
+    def _split_univariate_transformers(self, during_fit: bool = False):
+        """
+        Split univariate transformers into multiple transformers, one for each
+        column. This is useful to use the inherited `ColumnTransformer` class
+        parallelism.
+
+        Parameters
+        ----------
+        during_fit : bool, default=False
+            Whether the method is called during `fit_transform` (True) or
+            during `transform` (False). This is used to determine whether
+            to split the self.transformers_ attribute (when False) or the
+            self.transformers attribute (when True).
+        """
+        if during_fit:
```

Comment:
Nitpick: For consistency, it may be better to put the logic in a subfunction outside the scope of the class, the way you did with `_merge_transformers`. WDYT?